src/Mvc/Mvc.slnf

```diff
@@ -0,0 +1,132 @@
+﻿{
+  "solution": {
```

Comment:
We've been using slnf files in Blazor and MVC for a while now and personally I have been fairly happy with it.

In Mvc, we've had a smaller projection of project files that specifically include projects in the Mvc folder: https://github.com/dotnet/aspnetcore/blob/master/src/Mvc/MvcNoDeps.slnf. The big issue with a smaller projection is that VS expects referenced projects to have already been built. You'll get weird errors from MSBuild if referenced projects weren't already built. Usually I run a `build[cmd|sh]` on the cli before I launch a slnf and all's well with the world. On the upside, Smaller projections have a much faster dev inner-loop. On projects like MVC or Blazor which reference nearly everything in the repo, this makes a big difference. 

Going back to slnf files, there are a couple of annoying usage issues with it. 

* Adding additional projects to an slnf file isn't as simple as using the `Add Existing Project` dialog. I've typically ended up hand-editing it (not difficult since it's just JSON).
* CLI commands such as `dotnet build` \ `dotnet restore` etc do not work on slnf files, unlike sln files
* VS sometimes does not add slnf files to it's recently used project history. This one was aggravating enough that I filed a feedback issue - https://developercommunity.visualstudio.com/content/problem/660100/slnf-do-not-get-added-to-the-recent-projects-and-s.html.

As an aside, were you able to open the AspNetCore.sln file on a laptop? This was a while back, but I remember trying to build a large sln and it would take VS minutes before the UI became usable. It's rare you would want to open the root-level sln file, but it would be good to ensure it's not entirely unusable if you needed to open it.

Reply:
I was able to open it on my home desktop (which is an 16-proc Xeon with 32GB of RAM).  It wasn't pleasant until all the design time builds had run and been cached in the `.vs` folder, but after that it wasn't bad at all.

I haven't tried on my SurfaceBook 2 yet.

Reply:
> Adding additional projects to an slnf file isn't as simple as using the `Add Existing Project` dialog.

Does this apply to creating new projects as well. For example, will devs need to add a project to AspNetCore.sln then hand edit the relevant *.slnf files?

Reply:
Adding a project to a slnf file adds it to the backing sln. But I couldn't figure out what the gesture was to update the slnf file too while it was doing so.